sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQuerySinkMetrics.java

```diff
@@ -106,15 +106,15 @@ static Counter createRPCRequestCounter(RpcMethod method, String rpcStatus, Strin
    * <p>'RpcLatency-Method:{method};'
    *
    * @param method StorageWriteAPI method associated with this metric.
-   * @return Histogram with exponential buckets with a sqrt(2) growth factor.
+   * @return Histogram with exponential buckets with a size 2 growth factor.
    */
   static Histogram createRPCLatencyHistogram(RpcMethod method) {
     LabeledMetricNameUtils.MetricNameBuilder nameBuilder =
         LabeledMetricNameUtils.MetricNameBuilder.baseNameBuilder(RPC_LATENCY);
     nameBuilder.addLabel(RPC_METHOD, method.toString());
     MetricName metricName = nameBuilder.build(METRICS_NAMESPACE);
 
-    HistogramData.BucketType buckets = HistogramData.ExponentialBuckets.of(1, 34);
+    HistogramData.BucketType buckets = HistogramData.ExponentialBuckets.of(0, 17);
```

Comment:
can you add a comment on what the 0 and 17 is?

Reply:
done